.github/workflows/build.yml

```diff
@@ -1,19 +1,23 @@
 name: Deploy spec
 
-on: [push]
+on:
```

Comment:
https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/2

Reply:
This is very undesired; it's ideal to run on both push and PR, because CI should always run on pushed branches for which no pull request is made.